cookiecutter.json

```diff
@@ -5,6 +5,11 @@
       "title": "A Plone Project",
       "description": "Create a new Plone project with backend and frontend components"
     },
+    "project_classic": {
+        "path": "./project_classic",
+        "title": "A Plone Project with Classic UI",
+        "description": "Create a new Plone project with backend"
```

Comment:
A bit confusing... I'd be more especific, just by reading this I don't know what's the difference between both. I'd honestly go with the full product naming:
title: "A Plone 6 Classic UI project"
description: "Create a new Plone project that uses Classic UI" or "Create a new Plone project that uses Classic UI (Python only project)".

We should think out of the box as we have not been Plone developers for 20 years. 
If at the sprint there is such a profile, please show them the screen where the choices are shown.

Reply:
I went with "Create a new Plone project that uses Classic UI (Python only project)" for now but will try to find some more feedback tomorrow